rules/Transform/Rector/MethodCall/MethodCallToMethodCallRector.php

```diff
@@ -98,24 +98,26 @@ public function refactor(Node $node): ?Node
         }
 
         $isMethodCallCurrentClass = $this->isMethodCallCurrentClass($node);
+
+        if(!$node->var instanceof PropertyFetch && !$node->var instanceof Variable) {
```

Comment:
should back to original check.

```suggestion
        if (! $node->var instanceof PropertyFetch && ! $isMethodCallCurrentClass) {
```

as `Variable` can be `$foo` instead of `$this`.